GameServerLib/Scripting/CSharp/CSharpScriptEngine.cs

```diff
@@ -32,13 +40,24 @@ public bool LoadSubdirectoryScripts(string folder)
                 var trimmedPath = fileBasePath.Remove(0, basePath.Length);
                 var directories = trimmedPath.ToLower().Split(Path.DirectorySeparatorChar);
                 if (directories.Contains("bin") || directories.Contains("obj")) return false;
+                if (pathString.Contains("AssemblyInfo.cs")) return false;
                 return true;
             });
-            return !Load(new List<string>(allfiles));
+            if (allfiles.Count() == 0)
+            {
+                return CompilationStatus.NoScripts;
+            }
+            return Load(new List<string>(allfiles));
         }
 
-        //Takes about 300 milliseconds for a single script
-        public bool Load(List<string> scriptLocations)
+        /// <summary>
+        /// Loads scripts from a list of files and compiles them.
+        /// Takes about 300 milliseconds for a single script. Faster for a banch of scripts.
+        /// Returns true for success.
+        /// </summary>
+        /// <param name="scriptLocations"></param>
+        /// <returns>True means success</returns>
```

Comment:
What about false?

Reply:
I changed it to reflect the enum that gets returned. Good catch on making sure my documentation was accurate.